src/components/Calculator.js

```diff
@@ -16,29 +16,32 @@ class Calculator extends React.Component {
   render() {
     const { next, total } = this.state;
     return (
-      <div className="calc-container">
-        <div className="calc-display">
-          {next || total || 0}
```

Comment:
- [ ] OPTIONAL: I want to give you this optional comment which is not the requirement of this milestone. But it would improve your repo. When an operator button is clicked twice it throws an error on the browser. I think you can incorporate error handling to fix this problem. 
![Screenshot from 2021-08-25 15-39-41](https://user-images.githubusercontent.com/64894927/130767940-e9e5b8e5-1dae-4c80-925f-e1abe7b09523.png)